specificatie/openapi.yaml

```diff
@@ -20,7 +20,7 @@ info:
 paths:
   /ingeschrevenpersonen:
     get:
-      operationId: ingeschrevenNatuurlijkPersonen
+      operationId: getIngeschrevenNatuurlijkPersonen
```

Comment:
Kan dit worden hernoemd naar GetIngeschrevenPersonen? Dan is het ook consistent met BRK. Daar wordt voor operationId's PascalCase toegepast. 'Natuurlijk' eraf omdat dit verder nergens wordt gebruikt in de OAS